lib/lotus/entity.rb

```diff
@@ -144,13 +144,28 @@ def attributes(*attrs)
           self.attributes.merge attrs
 
           attrs.each do |attr|
-            attr_accessor(attr) if define_attribute?(attr)
+            attr_accessor_fabric(attr) if define_attribute?(attr)
```

Comment:
on second look, `define_attribute?` is a poor naming, what do you think about renaming it to `defined_attribute?`